Kleisli.md

```diff
@@ -0,0 +1,169 @@
+autoscale: true
+footer: ![Arrow](arrow-brand-128x128.png) [@47deg](https://twitter.com/47deg) :: [Λrrow](http://arrow-kt.io) :: [http://arrow-kt.io/docs/datatypes/kleisli/](http://arrow-kt.io/docs/datatypes/kleisli/)
+slidenumbers: true
+
+# Kleisli 
+
+__`Kleisli`__ is a data type used in __Λrrow__ to model a transformation 
```

Comment:
__`Kleisli`__ is a data type used in __Λrrow__ to model a a secuence of chained functions of the shape `(A) -> F<B>` where `A` is the result of a previously executed computation and `F<B>` represents any data type such as `DeferredK`, `IO`, `ObservableK`, `Option` etc. that has a type argument.